NxMapData.lua

```diff
@@ -12074,6 +13232,13 @@ Map.MiniMapBlks = {
 		Map.MapWorldInfo[1355].X + Map.MapInfo[12].X - 106.85, Map.MapWorldInfo[1355].Y + Map.MapInfo[12].Y - 295.44,
 		"World\\Minimaps\\nazjatar"
 	},	
+	[13] = {
+		Map.ShadowlandsBlks,
+		1416,
+		14, 16,
+		Map.MapWorldInfo[1550].X + Map.MapInfo[13].X + 594.95, Map.MapWorldInfo[1550].Y + Map.MapInfo[13].Y - 440,
```

Comment:
As stated in the PR, I just adjusted the constant X and Y offsets manually by changing these and reloading wow repeatedly. Maybe someone could take a look if these need some adjustment.

Reply:
I'm doing it the same way.